### PR TITLE
Evolution times quadrupled so I can make default evolution scaling factor 1

### PR DESCRIPTION
**Pull request by Inglonias**
#### Summary
Balance "Redid evolution so behavior is identical with a new default scaling factor of 1"

#### Purpose of change
Honestly, I was bored and was thinking about monster evolution. Figured this might kill a few minutes.

#### Describe the solution
Quadrupled all monster-related `half_life` json entries so I could set the default monster evolution scaling factor to 1.

#### Describe alternatives you've considered
Changing things behind the scenes so that an evolution scaling factor of 1 is identical to a scaling factor of 4 by simply quartering and quadrupling the number whenever it becomes relevant. This seems like a bad idea with the potential for unwanted behavior.

#### Testing
I am absolutely going to need to test this, and I absolutely have not yet done so. Please don't merge this until I get to test it, even if it does end up being a good idea, and I'm still not sure it's a good idea.

#### Additional context
Even if this does what I think it will do, this is a potentially bad idea because it breaks pretty much every mod outside this repo that adds monsters until they update, which is... maybe not the best idea. This was a spur-of-the-moment change.

### CODE REVIEW

**Comment by Inglonias** [comment]
I fully expect this to break some unit tests. I seriously doubt that the game is just going to sit there and let me change this many numbers without ruining something. I'll rebuild that bridge when it stops being on fire.

File: data/json/monsters/fungus.json [hunk 1]
```diff
@@ -327,7 +327,7 @@
     "harvest": "exempt",
     "special_attacks": [ [ "PLANT", 100 ] ],
     "death_function": { "message": "The %s disintegrates!", "corpse_type": "NO_CORPSE" },
-    "upgrades": { "half_life": 10, "into": "mon_fungaloid" },
+    "upgrades": { "half_life": 56, "into": "mon_fungaloid" },
```
Comment (Inglonias) [inline data/json/monsters/fungus.json:330]: bad math

File: data/json/monsters/insect_spider.json [hunk 2]
```diff
@@ -459,7 +459,7 @@
     "harvest": "arachnid_bee",
     "anger_triggers": [ "HURT", "FRIEND_DIED", "PLAYER_CLOSE" ],
     "fear_triggers": [ "FIRE" ],
-    "upgrades": { "half_life": 21, "into": "mon_bee_mega" },
+    "upgrades": { "half_life": 56, "into": "mon_bee_mega" },
```
Comment (Inglonias) [inline data/json/monsters/insect_spider.json:462]: bad math

**Comment by Inglonias** [comment]
> This also reduces the ability to make evolution artificially faster.

I don't see how - The behavior should still be the same, with the default value for balancing being one instead of four for whatever reason. I also can't tell if this means you think this is a good or bad idea.

**Comment by Maleclypse** [comment]
> > This also reduces the ability to make evolution artificially faster.
> 
> I don't see how - The behavior should still be the same, with the default value for balancing being one instead of four for whatever reason. Unless you're talking about the zombie masters having the ability to artificially cause monsters to evolve quickly, that is. I also can't tell if this means you think this is a good or bad idea.

I think he means possibly in the range of adjustment. If someone had increased evolution from 4 to 0.05 what would you need to do on a scale of zero to one for the equivalent speed up. I personally would have gone the other direction and normalized from four to ten as that would increase the available range of adjustments on speeding it up while not too badly impacting the scaling it down. But I’m not really a math guy so all of my assumptions here could be wildly wrong.

**Comment by Inglonias** [comment]
> I think he means possibly in the range of adjustment. If someone had increased evolution from 4 to 0.05 what would you need to do on a scale of zero to one for the equivalent speed up. I personally would have gone the other direction and normalized from four to ten as that would increase the available range of adjustments on speeding it up while not too badly impacting the scaling it down. But I’m not really a math guy so all of my assumptions here could be wildly wrong.

An equivalent speed with the default of 1.0 would be 0.0125, so you'd have to set things to 0.01 or 0.02 depending on how you felt. That said, you're making evolution 100x faster if you do that. You'd be reducing a 56 day half life to 13 and a half hours. The difference would be a matter of minutes in gameplay terms.

**Comment by Inglonias** [comment]
Thought about things a little more. Closing this PR on the grounds that a PR whose purpose is to change nothing with no real further goal behind it is really not helpful to anybody. Doesn't matter if this is a good idea or not. There's no point to doing it.

**Comment by anothersimulacrum** [comment]
The lowest value of 0.01 would be 4x slower than it would formerly be. How much that actually matters is questionable, but it's just a factor to consider.

**Pull request closed without merging**